serverless/functions/concepts.mdx

```diff
@@ -132,6 +140,17 @@ Serverless Functions simplify deploying applications to the Cloud. You are only
 
 <Concept>
 
+## SQS trigger
+
+An SQS (Simple Queue Service) trigger is a mechanism that connects a function to an [SQS queue](/serverless/messaging/concepts/#sqs) and invokes the function automatically whenever a message is added to the queue.
```

Comment:
Should the documentation link be #sqs-queue at the end or is this one correct?

Reply:
SQS queue does not exist as a concept. Applied the link to only "SQS" rather than "SQS queue"